src/app/shared/services/authentication-implementation.service.ts

```diff
@@ -2,21 +2,24 @@ import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
 import { HttpClient } from '@angular/common/http';
 import { BehaviorSubject, Observable } from 'rxjs';
-import { LocalStorage } from 'ngx-store';
+import { Store } from '@ngrx/store';
```

Comment:
Reorganize your code. It seems you can remove this service and use effect for login and store user info.